pkg/operator/encryption/encryptionconfig/config.go

```diff
@@ -92,6 +106,14 @@ func ToEncryptionState(encryptionConfig *apiserverconfigv1.EncryptionConfigurati
 				continue // should never happen
 			}
 
+			// enrich KeyState with values from secrets
+			for _, k := range backedKeys {
+				if state.EqualKeyAndEqualID(&ks, &k) {
+					ks = k
```

Comment:
okay, that's the spot where we add more info to the keys.

Reply:
yep. Everything else is basically plumbing update. Yes, I should have made two commits.